src/Form/DeliveryType.php

```diff
@@ -293,6 +299,7 @@ public function configureOptions(OptionsResolver $resolver)
             'with_weight' => true,
             'with_tags' => $this->authorizationChecker->isGranted('ROLE_ADMIN'),
             'with_dropoff_doorstep' => false,
+            'use_time_slots' => true,
```

Comment:
Also, there was a problem when a recurrence rule was set up using a pre-defined time slot. The time was saved correctly but was not properly displayed to a user